### ISSUE TITLE
Snackbar - Accessibility requirements

### ISSUE THREAD

**Issue by jessicawoodin** [department-of-veterans-affairs/va-mobile-library#155]
**To dos**
- [x] Research accessibility challenges of Snackbars
- [x] Determine if we need to make any improvements to our Snackbar
- [x] Review with Brea
- [x] Add recommendations to Snackbar epic

**Resources**
- https://www.magentaa11y.com/checklist-native/snackbar-toast/
- https://www.magentaa11y.com/native/
- https://github.com/uswds/uswds/discussions/5770
- https://wise.design/components/snackbar
- https://design.aetna.com/components/android/snackbar.html

**Recommendations**
- "Snackbar or toast must be dynamically announced by the screen reader without moving focus to it."
     - **Current**: App automatically moves the focus to the snackbar.
     - **Proposed**: Snackbar is announced, but focus is not moved.
- "If the Snackbar persists, it should be in the swipe order for the screen reader user and tab order for the keyboard user."
     - **Current**: Snackbar is read in between different pieces of the bottom tab bar.
     - **Proposed**: Snackbar should read after the last piece of content and before the bottom tab bar.
- Visual appearance
     - **Current**: Snackbar appears on top of the bottom tab bar.
     - **Proposed**: Snackbar should appear above the bottom tab bar, so it does not block the tab bar.
